resources/asciidoctor/spec/copy_images_spec.rb

```diff
@@ -0,0 +1,225 @@
+require 'copy_images/extension'
+require 'fileutils'
+require 'tmpdir'
+
+RSpec.describe CopyImages do
+  before(:each) do
+    Extensions.register do
+      tree_processor CopyImages
+    end
+  end
+
+  after(:each) do
+    Extensions.unregister_all
+  end
+
+  private
+  def copy_attributes copied
+    {
```

Comment:
Is the funky formatting meant to communicate something here?

Reply:
We're returning a map. Would it be clearer if I said `return {`? I don't ruby too well so I'm not sure.

Reply:
I think it would avoid confusion with things like block formatting. It makes the reader have to think about whether there are any `| |` present or something.